portal-web/test/functional/com/liferay/portalweb/macros/headless/journal/HeadlessWebcontentAPI.macro

```diff
@@ -542,7 +548,7 @@ definition {
 	macro returnAllstructuredContentId {
 		var response = HeadlessWebcontentAPI._allStructuredContent();
 
-		var structuredContentId = JSONUtil.getWithJSONPath("${response}", "$..id");
+		var structuredContentId = JSONUtil.getWithJSONPath("${response}", "$..['items'][?(@.creator)].id");
```

Comment:
I made these changes cause with `$..id` returning all the id, including the site id, I noticed when see thar log had some errors.